tools/isoplot/isoplot.xml

```diff
@@ -1,9 +1,14 @@
-<tool id='isoplot' name='Isoplot: Generate plots from isocor output' version='1.3.0+galaxy1' profile='20.01'>
+<tool id='isoplot' name='Isoplot: Generate plots from isocor output' version='@TOOL_VERSION@+galaxy2' profile='20.01'>
+    <macros>
+        <token name="@TOOL_VERSION@">1.3.1</token>
+    </macros>
     <requirements>
-        <requirement type='package' version='1.3.0'>isoplot</requirement>
+        <requirement type='package' version='3.0.3'>jinja2</requirement>
+        <requirement type='package' version='1.3.1'>zlib</requirement>

```

Comment:
Wondering if those should be added to the bioconda recipe (would save us 1 container). But it would slow down this PR.

Reply:
might remove zlib, not sure if that is really needed, but the jinja pin is needed here or in the bioconda package.

Reply:
> might remove zlib, not sure if that is really needed, but the jinja pin is needed here or in the bioconda package.

Your choice :)